shell/src/main/java/alluxio/cli/fsadmin/report/MetricsCommand.java

```diff
@@ -54,7 +54,6 @@ public MetricsCommand(MetaMasterClient metaMasterClient, PrintStream printStream
    */
   public int run() throws IOException {
     Map<String, MetricValue> metricsMap = new TreeMap<>(mMetaMasterClient.getMetrics());
-
     Set<String> operations = new HashSet<>();
     operations.add("DirectoriesCreated");
```

Comment:
To prevent issues like this in the future, could we use a common string constant here and on the server-side where the metric name is defined?